config/locales/en.yml

```diff
@@ -73,6 +73,7 @@ en:
         destroy_all_shifts:     "delete all shifts"
         destroy_all_users:      "delete all users"
         destroy_all_offsprings: "delete all offsprings"
+        export_to_csv:          "export to csv"
```

Comment:
Please don't use more titleize. I've found the usage to be wrong and now I am changing everything to direct text. Is it easier to control